app/services/processors/org_users_updater.rb

```diff
@@ -1,8 +1,14 @@
 module Processors
   class OrgUsersUpdater < BaseService
     def call
-      User.where(login: current_members).find_each { |user| user.update(company_member: true) }
-      User.where.not(login: current_members).find_each { |user| user.update(company_member: false) }
+      current_time = Time.current
+      User.where(login: current_members, company_member_since: nil).find_each do |user|
+        user.update(company_member_since: current_time)
```

Comment:
Do you have important validations here? Otherwise you could use `upsert_all` and execute just a query per batch.